data/provider/src/main/java/tech/pegasys/artemis/api/ChainDataProvider.java

```diff
@@ -134,12 +134,12 @@ public boolean isStoreAvailable() {
                   .getBestBlockRoot()
                   .orElseThrow(ChainDataUnavailableException::new);
           return combinedChainDataClient
-              .getStateAtSlot(slot, bestRoot)
+              .getBlockAndStateInEffectAtSlot(slot, bestRoot)
```

Comment:
so this is the important bit here, right?

Reply:
yep